File: Day3/Python/Ques10.py
l=int(input("Enter lower limit:"))
u=int(input("Enter upper limit:"))
print("Prime Numbers in range (",l,",",u,")")
for i in range(l,u+1):
    if i>1:
        is_Prime=True
        for j in range (2,i):
            if i%j==0:
                is_Prime=False
                break
        if(is_Prime):
            print(i)

File: Day3/C/Ques10.c
#include<stdio.h>
int main(){
    int u,l,prime;
    printf("Enter lower range:");
    scanf("%d",&l);
    printf("Enter upper range:");
    scanf("%d",&u);
    printf("Prime number in the range %d - %d :",l,u);
    for(int i=l;i<u;i++){
        for(int j=2;j<i;j++){
            if(i%j!=0){
                printf("%d  ",i);
                break;
            }
        }
    }
    return 0;

}